sphinx/builders/latex/transforms.py

```diff
@@ -159,3 +161,69 @@ def visit_footnote_reference(self, node):
     def unknown_departure(self, node):
         # type: (nodes.Node) -> None
         pass
+
+
+class BibliographyTransform(SphinxTransform):
+    """Gather bibliography entries to tail of document.
```

Comment:
In nearly future, we can add a new settings for the position of thebibliography.
This only gathers them to the tail of document. But some books lists them up on the tail of each chapter.